### DISCUSSION TITLE
zotero mac无法唤醒zotero-gpt

### DISCUSSION THREAD

**Discussion by Demo-Zhang** [MuiseDestiny/zotero-gpt#205] [Q&A] [answered]
已经安装了zotero-gpt插件，也输入了密钥，但是按shift+/无法唤醒gpt，按了之后没有反应。但是按shift+p可以唤醒zotero style，这是为什么呀？

**Comment by MuiseDestiny** [comment, marked as answer]
**Ctrl**

**建议使用插件前阅读其Readme**

**Reply by Demo-Zhang** [reply]
ctrl也试过了，无法唤醒T_T

**Reply by MuiseDestiny** [reply]
有没有按钮，gpt的按钮，点击一下可以。

**Reply by Demo-Zhang** [reply]
我刚用command试了一下，可以唤醒了。请问下开发者，怎么让chatgpt输出的是中文呢，我是用中文提问的，但是他给我提供的是英文😂😂😂

**Reply by MuiseDestiny** [reply]
你zotero是英文界面吧，改成中文就好了。

**Reply by Demo-Zhang** [reply]
哦，好的。谢谢啦。

**Comment by Stanleycn** [comment]
mac下的zotero安装gpt可以使用command+/唤起，但是为什么没有出现图标
